boards/ek-tm4c1294xl/src/main.rs

```diff
@@ -23,7 +23,7 @@ pub mod io;
 const NUM_PROCS: usize = 4;
 
 // How should the kernel respond when a process faults.
-const FAULT_RESPONSE: kernel::process::FaultResponse = kernel::process::FaultResponse::Panic;
+const FAULT_RESPONSE: kernel::FaultResponse = kernel::FaultResponse::Panic;
```

Comment:
We were doing `process.rs` both ways (a couple lines below this the array of processes is made with just `kernel::Process`), and I just picked one.

I think I went this approach in part because making the entire module public may expose too much.